### ISSUE TITLE
Can not find "Queue" in configuration menu

### ISSUE THREAD

**Issue by alexliju** [mautic/mautic#9052]
[//]: # ( Invisible comment: 
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Before you create the issue:
IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII
Search for similar report among other reported issues.
Learn how to troubleshoot at https://www.mautic.org/docs/en/tips/troubleshooting.html
Use drag&drop to attach images or other files )

## Bug Description
If we want to use RabbitMQ we have to configure it at configuration menu. But now there is no such menu to configure, so RabbitMQ can't be activated.

| Q   | A
| --- | ---
| Mautic version | 3.0
| PHP version | 7.3.19
| Browser | Chrome(Macbook)

### Steps to reproduce
1. Click configuration icon at right up corner.
2. Click "configuration"
 
### Log errors
No such errors

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )
![1596510512713](https://user-images.githubusercontent.com/58349066/89248684-e700db00-d642-11ea-9997-35c2df49a6d1.jpg)

**Comment by mautibot** [comment]
This issue has been mentioned on **Mautic Community Forums**. There might be relevant details there:

https://forum.mautic.org/t/page-hit-data-not-being-populated-from-query/16943/2

**Comment by TheNavigat** [comment]
I'm having the same problem on Mautic 3.1.2

**Comment by RCheesley** [comment]
Hi folks, I just checked in with the core team, and this was deprecated in 3.x with this commit: https://github.com/mautic/mautic/commit/74b8c3e942636a953b081a285c5b696563806662#diff-0c8e7cb890a2152fe377180fde4a86b3314a41f4933622578bfd53b599f26f29

> "Due to M3's containerless config, queue settings can no longer be configured through the UI due to its dependency on the container/cache. This now requires manual configuration in an instance's local configuration file."

I've asked the team if they could update the documentation accordingly to take this into account, but the answer to your question is that it needs to be configured in your local.php configuration file.  You can see the config required in the commit above.

**Comment by myupchar** [comment]
Yes, but it doesn't work and gives this error - https://github.com/mautic/mautic/issues/9284

It's a great software but it seems like Mautic suffers from poor documentation and newer versions are not tested thoroughly.

**Comment by PierreAmmeloot** [comment]
Hello @RCheesley is it possible to update the documentation with the new procedure?

Doc. link : https://docs.mautic.org/en/queue

**Comment by myupchar** [comment]
> Hello @RCheesley is it possible to update the documentation with the new procedure?
> 
> Doc. link : https://docs.mautic.org/en/queue

Challenge is that it contains a small blurb about 3.x but that fix doesn't really work

**Comment by RCheesley** [comment]
@PierreAmmeloot check the edit button at the bottom of the page. We have a PR here: https://github.com/mautic/mautic-documentation/pull/189 but we can't merge it until @maxlawton pulls in the changes as it's made on a private branch. Perhaps check that PR and see if it adds the details you expect. If not, make a new PR by clicking the edit button (on the bottom of every docs page).

**Comment by PierreAmmeloot** [comment]
Thanks @RCheesley for your feedback. I'm happy to modify the documentation. Except that the new procedure is not very clear. Do you know it?

@myupchar indeed if it doesn't work it's a problem. What bug do you have?
I am looking for a way to fix it. Without success for the moment. :-)

**Comment by myupchar** [comment]
This is what we are seeing - 

sudo php bin/console mautic:queue:process --env=prod -i page_hit

In Container.php line 348:

  You have requested a non-existent service "old_sound_rabbit_mq.mautic_consumer".

We are on v3.3.3

**Comment by RCheesley** [comment]
Folks if you have an issue please raise a new bug. Don't expect people to triage your issue here.

@PierreAmmeloot click edit at the bottom of the page. Edit the page in GitHub. Make a PR. It's quite easy really :)

if you prefer working with Git then just fork and clone github.com/mautic/mautic-documentation and make a PR in the usual way.

**Comment by myupchar** [comment]
Filed here - https://github.com/mautic/mautic/issues/10378 (for future reference)

This has been reported earlier as well but it went stale.

**Comment by gvaniya** [comment]
this issue are facing in 4.0.1
